package.json

```diff
@@ -26,11 +26,11 @@
     "clean": "rimraf ./dist && rimraf ./modules && rimraf ./es6",
     "clean:examples": "rimraf ./public",
     "build": "yarn clean && yarn test && yarn build:dist && yarn build:es5 && yarn build:es6",
-    "build:dist": "webpack --progress --profile --colors",
```

Comment:
`--colors` seems to be unsupported now

Reply:
I see `--color` instead in here https://webpack.js.org/api/cli/ But I guess we can remove it if this is defaulted to true.

Reply:
I see, I missed it indeed! I tested locally and for me it's enabled by default, though I can do --no-color to disable it. So I'll leave it out for now :-)